### PR TITLE
Added .travis.yml

### PR DESCRIPTION
**Pull request by zlangbert**
Not entirely sure what you need to do for travis, but here's a start

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+node_js:
+  - "4.2"
+
+sudo: false
+
+cache:
+  directories:
+    - node_modules
+
+script: npm run lint && npm run test
```
Comment (oliviertassinari) [inline .travis.yml]: Missing a last blank line

File: package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "description": "A React component for swipeable views",
   "main": "lib/index.js",
   "scripts": {
+    "lint": "eslint --ext .js .jsx .",
```
Comment (oliviertassinari) [inline package.json]: What is the ext option for?

Comment (oliviertassinari) [inline package.json]: It's linting the lib folder but shouldn't (e.g. /home/travis/build/oliviertassinari/react-swipeable-views/lib/index.js)

Comment (zlangbert) [inline package.json]: > What is the ext option for?

eslint only seems to lint `.js` by default, --ext gets both extensions

> It's linting the lib folder but shouldn't (e.g. /home/travis/build/oliviertassinari/react-swipeable-views/lib/index.js) 

Added a .eslintignore. Are we good with linting the docs?

**Comment by zlangbert** [comment]
Added travis-ci badge as well

**Pull request merged**